scripts/filemanager.js

```diff
@@ -9,8 +9,34 @@
  *	@author		Pavel Solomienko <https://github.com/servocoder/>
  *	@copyright	Authors
  */
-
+ 
 (function($) {
+  
+  // keep variables in non global object
+var fm = {};
```

Comment:
you are right. I initially thought it a good idea, to have one globalcontext, but I moved the fm-variable into function context, it would be just helpful to have it as a data holder object - to check all the variables, which FM is using and to easily switch context, of course. But if you think it´s better without (the reference chain is of course longer), I can remove it and update the PR.